Source/Types/OutPoint.swift

```diff
@@ -7,26 +7,26 @@
 import Foundation
 
 public struct OutPoint: Codable, Param {
-    public let blockHash: H256?
-    public let cell: CellOutPoint?
+    public let txHash: H256?
+    public let index: Number?
 
     enum CodingKeys: String, CodingKey {
-        case blockHash = "block_hash"
-        case cell
+        case txHash = "tx_hash"
+        case index
     }
 
-    public init(blockHash: H256? = nil, cell: CellOutPoint? = nil) {
-        self.blockHash = blockHash
-        self.cell = cell
+    public init(txHash: H256? = nil, index: Number = "0") {
```

Comment:
Could you double check if `txHash` and `index` are optional?

Reply:
I have fixed this issue.

https://github.com/cryptape/ckb-internal/issues/1264#issuecomment-521477216
https://github.com/nervosnetwork/ckb-sdk-ruby/blob/3919dcf008378bd2f36629358dffd750d20e018c/lib/ckb/types/out_point.rb